src/lib/y2partitioner/sequences/add_partition.rb

```diff
@@ -8,15 +8,29 @@
 
 module Y2Partitioner
   module Sequences
+    # Collecting params of partition to be created
+    # and remembering them across dialogs
+    class PartitionTemplate
```

Comment:
Yes, I did it this way because of the docs. [Struct](http://www.rubydoc.info/stdlib/core/Struct) does not say how to document the fields.